Doc/library/struct.rst

```diff
@@ -345,18 +351,20 @@ Examples
 ^^^^^^^^
 
 .. note::
-   All examples assume a native byte order, size, and alignment with a
+   Unless otherwise specified,
+   all examples assume a native byte order, size, and alignment with a
```

Comment:
I don't think we should state this note setting up "modal" expectations of what people should assume. How about just stating:

note::
   Native byte order examples (designated by the ``@`` format prefix or lack of a prefix) may not match what your own machine produces as that depends on the underlying platform.

Reply:
Eh, my mistake. I'll reword it.